### PR TITLE
chore(release): update changelog and bump version 0.20.1

### PR DESCRIPTION
**Pull request by antouhou**
## Issue being fixed or feature implemented

Release 0.20.1

## What was done?

- Update changelog
- Bump version to 0.20.0

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## [0.20.1](https://github.com/dashevo/dapi-client/compare/v0.19.2...v0.19.3) (2021-05-26)
+
+### Bug Fixes
+
+* crash when connection to older versions of dapi that don't send metadata ([46b36ae](https://github.com/dashevo/js-dapi-client/commit/46b36ae72743fd1cba8bf8cba73c36b1a350f4d7))
+
```
Comment (jawid-h) [inline CHANGELOG.md:6]: I believe it should be 3 empty lines

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## [0.20.1](https://github.com/dashevo/dapi-client/compare/v0.19.2...v0.19.3) (2021-05-26)
+
```
Comment (jawid-h) [inline CHANGELOG.md:2]: I believe it should be 2 empty lines

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+
+### Bug Fixes
+
+* crash when connection to older versions of dapi that don't send metadata ([46b36ae](https://github.com/dashevo/js-dapi-client/commit/46b36ae72743fd1cba8bf8cba73c36b1a350f4d7))
```
Comment (shumkov) [inline CHANGELOG.md:6]: Please links PR instead of commit

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
+## [0.20.1](https://github.com/dashevo/dapi-client/compare/v0.19.2...v0.19.3) (2021-05-26)
```
Comment (shumkov) [inline CHANGELOG.md:1]: Link is incorrect

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,12 @@
+## [0.20.1](https://github.com/dashevo/dapi-client/compare/v0.20.0...v0.20.1) (2021-05-26)
+
+
+### Bug Fixes
+
+* * crash when connecting to an older version of dapi ([#252](https://github.com/dashevo/dapi-client/issues/252))
```
Comment (shumkov) [inline CHANGELOG.md:6]: Two stars. Also, I guess we have two fixes: one for platform endpoints and another one for getTransaction what we forgot to do initially. I such cases we usually list several PRs for one item.

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**